spec/routes/web/project/postgres_spec.rb

```diff
@@ -288,16 +287,6 @@
         expect { find "#fwr-delete-#{pg.firewall_rules.first.ubid} .delete-btn" }.to raise_error Capybara::ElementNotFound
       end
 
-      it "can not delete firewall rules if not exist" do
```

Comment:
Because, we don't return 404 anymore. 404 was already handled as success in the front end, so we have opportunity to simplify the logic.

I added commit message.